vmigration_helper/helpers/command.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+
+from django.core.management import BaseCommand
+from django.db import DEFAULT_DB_ALIAS
+
+
+class MigrationCommand(BaseCommand, ABC):
```

Comment:
Common options for all our migration commands

Reply:
Nice clean-up of the dupe from #14.